merlin/dag/executors.py

```diff
@@ -38,7 +38,7 @@ class LocalExecutor:
 
     def transform(
         self,
-        df,
+        transformable,
         nodes,
         output_dtypes=None,
```

Comment:
similar to my question about `additional_columns` in that it's not strictly related to this change since it was here already and looks like mostly to match the interface of the `DaskExecutor`. However, it appears to be unused, wondering if it can be removed (in future change potentially). Or is it here as a placeholder to be used for something?

Reply:
I think it's there to make the method signatures match across executors, since Dask uses it to give you partitions with the correct/user-specified dtypes in the output dataframe. How we want to handle arguments that some executors use and others don't was an open question, so I made the signatures match as a hopefully sensible default. This will probably become clearer if/when we try to add a third executor—possibly an async version of `LocalExecutor` or a distributed executor based on Ray.